tests/logictest/suites/base/03_dml/03_0016_insert_into_values

```diff
@@ -52,6 +52,15 @@ create table if not exists st1(a string);
 statement error 1303
 insert into st1 values ('a', 'b');
 
+statement ok
+insert into st1 values (upper('a'));
+
+statement ok
+set insert_values_disable_expression = 1;
```

Comment:
```suggestion
set global insert_values_disable_expression = 1;
```

Reply:
We should add a test to check `insert_values_disable_expression = 1` works:
```
insert into st1 values ('a'),('b'), ('c');
select * from st1;
```